prime-router/src/main/kotlin/azure/TokenFunction.kt

```diff
@@ -0,0 +1,68 @@
+package gov.cdc.prime.router.azure
```

Comment:
This is the new endpoint used to send a privately signed JWT, so you can get back a token valid for 5 mins to submit reports.